apiv2/apiv2_utils.py

```diff
@@ -52,7 +52,7 @@
 from utils.similarity_utilities import api_search as similarity_api_search
 from utils.similarity_utilities import get_sounds_descriptors
 
-logger_error = logging.getLogger("api_errors")
+error_logger = logging.getLogger("api_errors")
```

Comment:
in `apiv2/exceptions.py` this is called api_errors_logger

Reply:
fixed in c9a63184c9ef42bd624493321a4d7d32e991f917